src/things.ts

```diff
@@ -112,6 +115,8 @@ async function getTasksFromThingsDb(
         ON TMTag.uuid = TMTaskTag.tags
     LEFT JOIN TMArea
         ON TMTask.area = TMArea.uuid
+    LEFT JOIN TMTask TMProject
```

Comment:
Thanks! that makes sense. My idea was to be able to group tasks by area/project/heading levels. Will see if I get somewhere with this but keep an eye on this PR to check if you manage to add support for headings first :D

Thanks a lot for this contribution. Even if it does not get merged, Im pretty sure it will be very useful to lots of people like myself